app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Tasks</h1>
+
+<ul>
+    <% @tasks.each do |task| %>
+        <li>
+            <%= link_to task.name, url_for([@project, task]) %>
+        </li>
+     
+    <% end %>
+</ul>
```

Comment:
There should be a table like the projects. With "Show", "Edit", "Destroy" buttons.